git-ftp

```diff
@@ -752,8 +752,14 @@ submodule_catchup() {
 }
 
 set_submodule_args() {
-	# Duplicate the current required parameters
-	args=(--user "$REMOTE_USER" --passwd "$REMOTE_PASSWD")
+	# Duplicate the current required parameters if set
+	args="" 
+	if [ -z "$REMOTE_USER" ]; then
```

Comment:
Hum weird, it worked fine for me on a server with password in .netrc.
But you're absolutely right it should be "` -n `" i'll test again and keep you posted